docs/getting-started.md

```diff
@@ -0,0 +1,208 @@
+2. All links should the focus style working
+3. Use the right elements. Links should always be `<a>` and actions should be `<button>`
+#### :tada: GOOD link example
+```html
+<a href="post.php?post=632" aria-label="More on Using Meaningful Link Text">More...</a>
+```
+:::tip
+Would be even better if you could just avoid using non-informative link phrases(i.e."See more", "Click here") as link text.
+:::
+
+#### :lady_beetle: BAD link example
+```html
+<a href="post.php?post=632">
+    <img src="cat.svg" aria-label="cat svg here">
+    More...
+</a>
+```
+::: warning
+If the content of a link includes an img element(icon.etc), check if the icon actually needs the aria-label, the aria-label should also be descriptive
+:::
+### Buttons
+Buttons are fairly straightforward, ensure that: 
+- Check incorrect `aria-role` has been applied to any buttons.
+- Check context has been applied to the action/button e.g. `aria-label="my contextual text".`
+- `aria-controls` should be added to the button if it controls another component. (i.e. opening nav)
+
+#### :tada: GOOD button example
+```html
+<button aria-label="open navigation" aria-controls="nav" aria-expanded="false">Menu</button>
+```
+:::info
+When the nav is opened, `aria-exapanded` should be true instead
+:::
+#### :lady_beetle: BAD button example
+```html
+<button><a href="hi.html">Click me!</a></button>
+<div>I am a button!</div>
+```
+:::warning
+`<button>` should not be used for links. Also try to avoid styling `<div>` as button, as you would need to do a lot of extra work to ensure that it's accessible(see resources button vs div)
+:::
+
+### Navigation
+Important thing to remember for navigation is that we should also ensure that keyboard shortcuts work for controlling the menu e.g. `esc` key `closes`, `space` key `opens` etc.
+
+If a button controls the open state of the navigation, the button should have `aria-expanded` attribute (see Button example above)
+#### :tada: GOOD Navigation example
+```vue{1,6}
+<nav id="nav" role="navigation" aria-label="main">
+    <ul>
+      <li><a href="#">Some navigation</a></li>  
+    </ul>
+</nav>
+<secondary-nav id="secondary-nav" role="navigation" aria-label="secondary">
+    <ul>
+      <li><a href="#">Some navigation</a></li>
+    </ul>
+</seconadry-nav>
+```
+:::tip
+We should use `aria-label` to help user to distinguish the different nav available, when using `aria-label` on a <a href="https://web.dev/use-landmarks/">landmark element(i.e. nav)</a>, don't include the word role of the landmark, as screen readers will annouce it afterwards.
+:::
+
+#### :lady_beetle: BAD Navigation example
+```html
+<nav id="nav">
+    <ul>
+        <li><a href="#">Some navigation</a></li>
+    </ul>
+</nav>
+```
+:::info
+`role=”navigation”` is not required on explicit tags like `<nav>`. However older screen readers that do not support HTML5 tags will require it.
```

Comment:
If thats the case could we axe this example? I think we shouldn't need to worry about old browsers in this regard